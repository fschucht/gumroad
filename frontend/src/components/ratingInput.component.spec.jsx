import renderer from 'react-test-renderer';
import { RatingInput } from './ratingInput.component';

describe('RatingInput', () => {
  it('should render a RatingInput component', () => {
    const renderedComponent = renderer.create(<RatingInput currentRating={0}>Hello World!</RatingInput>).toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})