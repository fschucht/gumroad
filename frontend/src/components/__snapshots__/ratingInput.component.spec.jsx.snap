// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingInput should render a RatingInput component 1`] = `
<div
  className="rating-input"
>
  <p
    className="text"
  >
    Your Rating
  </p>
  <div
    className="rating-input__stars"
  >
    <svg
      className="rating-input__star"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      star.svg
    </svg>
    <svg
      className="rating-input__star"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      star.svg
    </svg>
    <svg
      className="rating-input__star"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      star.svg
    </svg>
    <svg
      className="rating-input__star"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      star.svg
    </svg>
    <svg
      className="rating-input__star"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      star.svg
    </svg>
  </div>
</div>
`;
